elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -186,6 +189,17 @@ def _record_gradients(self, batch_embedding, ids):
         return batch_embedding
 
     def call(self, input):
+        if (
+            self.tape
+            and not tf.executing_eagerly()
+            and not self._embedding_and_ids_graph
+        ):
+            raise RuntimeError(
+                "Call `init_for_graph_mode` for each ElasticDL embedding"
+                "layer before training model in graph mode.(i.e. under"
```

Comment:
Move the period in "mode." to the end of this sentence or just remove this period